src/pages/chatSelected/modules/conversation/index.ts

```diff
@@ -0,0 +1,107 @@
+import Block from '../../../../modules/block';
+import template from './conversation.tmpl';
+
+import ConversationInfo from './components/conversationInfo';
+import Avatar from '../../../../components/avatar';
+import Button from '../../../../components/button';
+import Input from '../../../../components/form/input';
+import Form from '../../../../components/form';
+import Message from './components/message';
+import ConversationActions from './components/conversationActions';
+
+import { IConversation } from './types';
+import infoIcon from '../../../../../static/icons/infoIcon';
+import optionsIcon from '../../../../../static/icons/optionsIcon';
+import userAvatar from '../../../../../static/images/userAvatar.png';
+import { conversationData, validationSchema } from './utils';
+import handleSubmit from '../../../../utils/functions/handleSubmit';
+import handleInputChange from '../../../../utils/functions/handleInputChange';
+
+export class Conversation extends Block {
+  constructor(props: IConversation) {
+    super(template, props);
+  }
+
+  render() {
+    const { topBar, messages, bottomBar } = this.props;
+
+    return this.compile({
+      topBar,
+      messages,
+      bottomBar,
+    });
+  }
+}
+
+export function ConversationModule(): Conversation {
+  const infoButton = new Button({
+    type: 'button',
+    content: infoIcon,
+  });
+
+  const topBar = new ConversationInfo({
+    avatar: new Avatar({
+      source: userAvatar,
+    }),
+    username: 'test',
+    button: infoButton,
+  });
+
+  const messages = conversationData.map(
+    ({ own, content, status, time }) =>
+      new Message({
+        own,
+        content,
+        status,
+        time,
+      }),
+  );
+
+  const optionsButton = new Button({
+    type: 'button',
+    content: optionsIcon,
+  });
+
+  const messageField = [
+    {
+      input: new Input({
+        name: 'message',
+        placeholder: 'Write a message...',
+        type: 'text',
+      }),
+    },
+  ];
+
+  messageField[0].input.setProps({
```

Comment:
Массивы стоит называть в множественном числе для читаемости
```suggestion
  messageFields[0].input.setProps({
```